blog/2024-02-28-reconciling-with-state.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: "reconciling-with-state"
+title: "Reconciling with State: Simplifications and Improvements to Better Support Framework-Specific SDKs"
+date: 2024-02-28
+authors: [toddbaert]
+description: "Simplifications and Improvements to Better Support Framework-Specific SDKs"
+tags: [client, spec, specification, web, mobile, sdk, state, events, react]
+draft: false
+---
+
+## Recent Specification Changes 🗒
+
+There's no shortage of frameworks available when it comes to the development of enterprise software.
+Weather it's the "frontend" or "backend", developers are quick to avail things like React, Spring, Gin and Flask to avoid boilerplate code and structure their applications in a familiar way.
```

Comment:
```suggestion
Whether it's the "frontend" or "backend", developers are quick to avail things like React, Spring, Gin and Flask to avoid boilerplate code and structure their applications in a familiar way.
```